src/components/menu/MenuHeader.scss

```diff
@@ -0,0 +1,19 @@
+@import '../../styles/variables';
+
+.bdl-MenuHeader {
+    position: absolute;
```

Comment:
What ancestor element are we positioning this to? Can we rely on that position?

Reply:
Yes, since MenuHeader is used only in full-screen mode. I moevd it into the DropdownMenu.scss instead to restrict it to that particular component

Reply:
Is there a use for `MenuHeader` outside of `DropdownMenu`?

In other words, if this component is specific to `DropdownMenu`, should we move it inside of that folder?

Reply:
At the moment, I'm not 100% sure where else we will be making changes. ContextMenu which is used in a similar context to DropdownMenu in EUA also uses Menu and may need the MenuHeader. This is something we might look at next quarter.